e2e/global-setup.ts

```diff
@@ -0,0 +1,7 @@
+import { test } from '@playwright/test';
+
+test('Login', async ({ page }) => {
+  await page.goto('http://localhost:8000/');
```

Comment:
@Fahdkassim Lets use the BaseUrl env variable here instead of hardcoding localhost.

Reply:
Done